dorcbot/dorcbot.py

```diff
@@ -49,34 +57,40 @@ def send(self):
 
 
 def error(description):
-    payload = Payload(fmt = '')
-    payload.embed=discord.Embed(color=0xFF5733, title='Error:')
+    payload = Payload(fmt='')
+    payload.embed = discord.Embed(color=0xFF5733, title='Error:')
     payload.embed.description = description
     payload.embed.set_thumbnail(url='https://c.tenor.com/OxvVRFnPZO8AAAAM/error-the-simpsons.gif')
 
     return payload
 
 
 def calldata(callsign):
-    callsign = callsign.split(';')[0]
-    pref = '{http://xmldata.qrz.com}'
-    login = requests.get(f'https://xmldata.qrz.com/xml/current/?username={QRZ_API_USER};password={QRZ_API_PASS};agent=q5.0')
-    key = ElementTree.fromstring(login.content)[0][0].text
-    resp = ElementTree.fromstring(requests.get(f'https://xmldata.qrz.com/xml/current/?s={key};callsign={callsign}').content)
-    data = resp.find(pref + 'Callsign')
-    if data:
-        calldata = {x.tag.split('}')[1]: x.text for x in data}
```

Comment:
I'll change this to use the xmltodict library, the code below is more readable and this dict comprehension duplicates what that code does.